packages/vite/src/utils/generator-utils.ts

```diff
@@ -208,13 +209,21 @@ export function addOrChangeTestTarget(
 ) {
   const project = readProjectConfiguration(tree, options.project);
 
+  // vitest coverage starts at the project root
+  // so this will output to projectRoot/coverage/*
+  // TODO: if we want workspaceRoot then we'd need to actually set it to the workspaceRoot path
+  // since those vars are not expected for executor options
+  // this is where jest outputs, but not required to do the same.
+  const coveragePath = joinPathFragments(
+    // workspaceRoot,
+    'coverage',
+    project.root === '.' ? options.project : project.root
+  );
```

Comment:
if we want to put the coverage reports at the workspaceRoot, then we'll have to put the actual workspace root path into the option (or expand that value in the executor manually) otherwise it'll be the folder name. 

this is kind of ugly IMO so would make sense to leave the coverage reports in the project root and make sure they are included in the task inputs

Reply:
or it could just be a relative path instead of the full workspaceRoot. will go with that instead